package.json

```diff
@@ -2,14 +2,15 @@
   "title": "Shield Recipe Client",
   "name": "shield-recipe-client",
   "id": "shield-recipe-client@mozilla.org",
-  "version": "0.1.0",
+  "version": "0.1.3",
   "description": "Client to download and run recipes for SHIELD, Heartbeat, etc.",
   "main": "index.js",
   "author": "Mike Cooper <mcooper@mozilla.com>",
   "repository": "https://github.com/mozilla/normandy-addon",
   "scripts": {
     "test": "jpm test",
-    "build": "jpm xpi"
+    "webpack": "node_modules/.bin/webpack data/consent-page.js data/dist/consent-page.js",
```

Comment:
npm scripts set up the path nicely, so you don't have to do `node_modules/.bin/`, you can just say `webpack`.